inetum-realjobs-frontend/src/app/components/resume/details/experience-details/experience-details.component.ts

```diff
@@ -0,0 +1,32 @@
+import {Component, OnInit, Input, Output, EventEmitter} from '@angular/core';
+import {Experience} from "../../../../models/experience";
+
+@Component({
+  selector: 'app-experience-details',
+  templateUrl: './experience-details.component.html',
+  styleUrls: ['./experience-details.component.scss']
+})
+export class ExperienceDetailsComponent implements OnInit {
+
+  @Input() experience: Experience;
+
+  @Input() deleteButton: boolean;
+  @Output() deleteEvent = new EventEmitter<number>();
+
+  constructor() {
+  }
+
+  ngOnInit(): void {
+  }
```

Comment:
Empty `ngOnInit` should be removed. Don't forget to remove `implements OnInit` for the class.